Admin/Update-Engines.ps1

```diff
@@ -241,22 +241,27 @@ foreach ($p in $Platforms) {
             $fullPkgUrl = $engineUrl + $manifest.ManifestFile.Package.version + "/" + $manifest.ManifestFile.Package.FullPackage.name
             $fullPkgPath = ($fullPkgDir + $manifest.ManifestFile.Package.FullPackage.name)
 
-            $wc.DownloadFile($fullPkgUrl, $fullPkgPath)
+            if (((Test-Path $fullPkgPath) -ne $true) -or ((Get-Item $fullPkgPath).Length -ne $manifest.ManifestFile.Package.FullPackage.Size)) {
```

Comment:
Is the `$fullPkgPath` always going to be different for each version?

Reply:
yes

![image](https://github.com/user-attachments/assets/b62402ea-42af-4673-b32e-dbeb10422079)

Reply:
todo: add some cleanup logic to the script...